docs/Getting Started/Finding Supported Features/index.md

```diff
@@ -0,0 +1,27 @@
+# Finding Supported Features
```

Comment:
"Determining Supported Features"?

"Checking Supported Features"?

Reply:
Fixed. Changed section name to "Checking Supported Features"